fusion_toolbox/reaction_rate/bosch_hale.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import sys
```

Comment:
```suggestion
```
You can remove this import once you replace the `sys.exit` with an appropriate error